openwebstart/src/main/java/com/openwebstart/os/MenuAndDesktopEntryHandler.java

```diff
@@ -87,7 +87,7 @@ private void updateEntries(final MenuAndDesktopEntriesFactory factory, final JNL
     }
 
     private void addEntries(final MenuAndDesktopEntriesFactory factory, final JNLPFile jnlpFile, final boolean addMenu, final boolean addDesktop) {
-        Executors.newSingleThreadExecutor().execute(() -> {
+        getDaemonExecutorService().execute(() -> {
```

Comment:
It won't be a problem to have this one as a non daemon executor. I prefer to change as less behaviour as possible with this PR so we should change it here to non daemon.